app/src/main/res/layout/activity_main.xml

```diff
@@ -27,6 +27,16 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/server_button" />
 
+    <EditText
```

Comment:
Missing `autofillHints` attribute - just add it:

https://developer.android.com/reference/androidx/autofill/HintConstants#AUTOFILL_HINT_USERNAME()